### ISSUE TITLE
Include metric of resource utilization in Compute Engine's window

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#2545]
Many users believe that if you just add more processors to VisIt's engine, it will *always* have the effect of making VisIt go faster.

However, in general, VisIt does not re-decompose the input data. It is able to process in parallel on a number of processors that is not more than the number of parts the input database is block into. For image and *some* netcdf files, I believe VisIt can re-decompose those on the fly, but that is the exception rather than the rule.

Although making VisIt warn users about possible under-utilization is undesireable, we can provide information in the Compute Engine's window from which they can make that determination for themselves.

This ticket is to enhance Compute Engine window with information on processor utilization of past (maybe past several) executes. What percent were idle, maybe what memory usage is across them, maybe even load balance. Heck, how about a VisIt plot that displays load balance of most recent engine executes?

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Mark Miller
Original creation: 02/09/2015 12:53 pm
Original update: 02/17/2015 04:21 pm
Ticket number: 2137
